website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -34,13 +34,13 @@ function**.
 If the network in which your Gateway is deployed applies egress filtering,
 you'll need to make sure the following outbound traffic is allowed:
 
-| Host                         | (IP Address)         | Port(s)       | Protocol(s)     | Purpose                    |
-| ---------------------------- | -------------------- | ------------- | --------------- | -------------------------- |
-| api.firezone.dev             | `34.102.202.25`      | `443`         | HTTPS/WebSocket | Control Plane API (IPv4)   |
-| api.firezone.dev             | `2600:1901:0:620b::` | `443`         | HTTPS/WebSocket | Control Plane API (IPv6)   |
-| N/A                          | Varies               | `3478`        | STUN            | STUN protocol signaling    |
-| N/A                          | Varies               | `49152-65535` | TURN            | TURN protocol channel data |
-| github.com, www.firezone.dev | Varies               | `443`         | HTTPS           | Gateway upgrades           |
+| Host                         | IP Address                                                                          | Port(s)       | Protocol(s)     | Purpose                                                         |
+| ---------------------------- | ----------------------------------------------------------------------------------- | ------------- | --------------- | --------------------------------------------------------------- |
+| api.firezone.dev             | `34.102.202.25`                                                                     | `443`         | HTTPS/WebSocket | Control Plane API (IPv4)                                        |
+| api.firezone.dev             | `2600:1901:0:620b::`                                                                | `443`         | HTTPS/WebSocket | Control Plane API (IPv6)                                        |
+| N/A                          | {/* Update this when Relay regions change */} See [relay-ips.json](/relay-ips.json) | `3478`        | STUN            | STUN protocol signaling                                         |
+| N/A                          | {/* Update this when Relay regions change */} See [relay-ips.json](/relay-ips.json) | `49152-65535` | TURN            | TURN protocol channel data                                      |
```

Comment:
The relay regions come from this Terraform file, right? https://github.com/firezone/firezone/blob/926e26f5788ab21d1c26478f271328795d23e0ce/terraform/environments/production/relays.tf

The "update when" comments should be in that file if that's the source of truth. It'll be easy to forget where we've put the comments in the docs if they're in the artifact side only.

`git grep` says this file should also be linked from `relays.tf`: https://firezone-git-docs-ip-ranges-firezone.vercel.app/kb/architecture/tech-stack#regional-availability